Packs/AzureResourceGraph/Integrations/AzureResourceGraph/AzureResourceGraph_description.md

```diff
@@ -0,0 +1,25 @@
+Azure Resource Graph is an Azure service designed to extend Azure Resource Management by providing efficient and performant resource exploration with the ability to query at scale across a given set of resources.
+
+Full documentation for this integration is available in the [reference docs](https://xsoar.pan.dev/docs/reference/integrations/azure-resource-graph).
+
+
+## Authorize Cortex XSOAR for Azure Log Analytics - Self-Deployed Configuration
+To use a self-configured Azure application, you need to add a new Azure App Registration in the Azure Portal. To add the registration, go to the [Microsoft article](https://docs.microsoft.com/en-us/azure/active-directory/develop/quickstart-register-app).
+
+### Required permissions
```

Comment:
please add the required permissions:)

Reply:
It says `user_impersenation` however I did not need it to perform REST calls and it is not very user friendly to use if it it was required.

Reply:
It says `user_impersenation` however I did not need it to perform REST calls and it is not very user friendly to use if it it was required.